src/web/history.rkt

```diff
@@ -81,6 +81,18 @@
       (core->tex core #:loc (and loc (cons 2 loc)) #:color "blue")
       "ERROR"))
 
+(define (mixed->fpcore expr ctx)
+  (define expr*
+    (let loop ([expr expr])
+      (match expr
+        [(? symbol?) expr]
+        [(? number?) expr]
+        [(? literal?) (literal-value expr)]
+        [`(if ,cond ,ift ,iff) `(if ,(loop cond) ,(loop ift) ,(loop ift))]
+        [`(,(? impl-exists? impl) ,args ...) `(,(impl->operator impl) ,@(map loop args))]
```

Comment:
But this won't desugar things like `cosd` right? Is that good?

Reply:
It won't which I think is fine because it'll display as `cosd`.